src/qtools/rad.cpp

```diff
@@ -301,8 +301,8 @@ void GetFaceExtents(Bsp* bsp, int facenum, int mins_out[2], int maxs_out[2])
 
 	f = &bsp->faces[facenum];
 
-	mins[0] = mins[1] = FLT_MAX;
-	maxs[0] = maxs[1] = FLT_MIN;
+	mins[0] = mins[1] = FLT_MIN;
+	maxs[0] = maxs[1] = FLT_MAX;

```

Comment:
This was correct before. To find the max of something, it starts out assuming the smallest possible value, then increases as it finds bigger values.

Reply:
Yes)) sorry)

Reply:
fixed)